### ISSUE TITLE
Missing config?

### ISSUE THREAD

**Issue by web64** [caneco/artisan-erm#1]
I'm getting this error when installing the package:

```bash
PHP Warning:  Uncaught ErrorException: require(/home/vagrant/code/catalyzer/vendor/caneco/artisan-erm/src/../config/erm.php): failed to open stream: No such file or directory in /home/vagrant/code/catalyzer/vendor/laravel/framework/src/Illuminate/Support/ServiceProvider.php:59
```

The file `config/erm.php` seems to be missing.

Looking forward to using this package! :)

Cheers,
Olav

**Comment by tigpt** [comment]
Yap, +1

```
TigPT:freebooting tiagorodrigues$ composer require caneco/artisan-erm
    1/12:	http://repo.packagist.org/p/provider-archived$6f2d2c9c548ca2ce0c8a91e593fc6bbdbba7090fd187aaf5f0ca7610d1f56c9e.json
    2/12:	http://repo.packagist.org/p/provider-latest$f181631962ad2dd8beebc42d401b0b015254b099815711786669449f019f84c2.json
    3/12:	http://repo.packagist.org/p/provider-2018$933b66460de7ffcc273b5d6fd8397c2f2baad31fc2d405e0b94ef6c201ba08f2.json
    4/12:	http://repo.packagist.org/p/provider-2018-04$2ea12a1bdcb16c0e06bf3d4fda5740c17134d41103f4c5df90d5dd4e0953e987.json
    5/12:	http://repo.packagist.org/p/provider-2018-07$fdd8bb8a567c4bb516f6ab1e246fbe81b031efdc2cda46384dd627299a415ff7.json
    6/12:	http://repo.packagist.org/p/provider-2013$d141de9c338d41d7f719371cc200ec942845b634bec419dfe1c2f842540c1ade.json
    7/12:	http://repo.packagist.org/p/provider-2019-01$376c45b009c106126722adea1c305ced9f447c7621f3193312dab1aa6178869c.json
    8/12:	http://repo.packagist.org/p/provider-2016$70effa6119f0d2ddd6baf970ecab39ecd5ec79767d0507d3df9172d1e05d7a7f.json
    9/12:	http://repo.packagist.org/p/provider-2014$8f5fa12ba559317930e072036b06c36a962edb7d84e0f3e02f48f7d0e0338a66.json
    10/12:	http://repo.packagist.org/p/provider-2017$8942d6ad958766656b13f918fb2770931420c26ac5459f7003fb599e10440734.json
    11/12:	http://repo.packagist.org/p/provider-2015$659e1a0c10a7467642ba2d83810f08334cb92ce212bb46496372f2028c1f91a1.json
    12/12:	http://repo.packagist.org/p/provider-2018-10$29c6255c985cf07bcdf4a6188a7ab1131bd58809aeda1170fd943516640794ce.json
    Finished: success: 12, skipped: 0, failure: 0, total: 12
Using version dev-master for caneco/artisan-erm
./composer.json has been updated
    1/1:	http://repo.packagist.org/p/provider-latest$64cceb3769f7ef655ba5ef16c1299c30067511696e0ca79ef344cd6b5061c0fd.json
    Finished: success: 1, skipped: 0, failure: 0, total: 1
Loading composer repositories with package information
Updating dependencies (including require-dev)
    1/1:	https://codeload.github.com/caneco/artisan-erm/legacy.zip/93c3dce29efaca83fac0479db6011dddf29b3b0d
    Finished: success: 1, skipped: 0, failure: 0, total: 1
Package operations: 1 install, 0 updates, 0 removals
  - Installing caneco/artisan-erm (dev-master 93c3dce): Cloning 93c3dce29e from cache
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

In ServiceProvider.php line 59:
                                                                               
  Illuminate\Support\ServiceProvider::mergeConfigFrom(): Failed opening requi  
  red '/Users/tiagorodrigues/laravel/freebooting/vendor/caneco/artisan-erm/sr  
  c/../config/erm.php' (include_path='.:')                                     
                                                                               

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255
```

**Comment by caneco** [comment]
Erm…I should have tried to install it before mention the package. Sorry about that.

Either way, thank you guys for the heads-up!
TL;DR, the `erm.php` config file was **"gitignored"** by mistake… It should be fixed now.
